app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt

```diff
@@ -179,9 +179,8 @@ open class MainActivity : AppCompatActivity(),
         private val VIDEOPRESS_2 = "[wpvideo OcobLTqC w=640 h=400 autoplay=true html5only=true3]"
         private val QUOTE_RTL = "<blockquote>לְצַטֵט<br>same quote but LTR</blockquote>"
         private val MARK = "<p>Donec ipsum dolor, <mark style=\"color:#ff0000\">tempor sed</mark> bibendum <mark style=\"color:#1100ff\">vita</mark>.</p>"
-        private val MARK_EXAMPLE = "<p>Hello <mark style=\"background-color:rgba(0, 0, 0, 0);color:#151853\" class=\"has-inline-color has-tertiary-color\"></mark></p>"
 
-/*        private val EXAMPLE =
+        private val EXAMPLE =
```

Comment:
Nit: what do you think about squashing this commit with [the one that introduced the commenting](https://github.com/wordpress-mobile/AztecEditor-Android/pull/943/commits/5fb3c5dedaaf3e1e1f1459a89999f9b626ef6153#diff-f560a91308e1ade3461de1cb5c543283d28801832ecf0a1475be13ce4eae23ecR184) to avoid adding extra steps to [the git-blame history](https://github.com/wordpress-mobile/AztecEditor-Android/blame/688a5bb8b10525f051ebd405af65a0818dd8a32b/app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt#L183)? No strong feeling on this.

Reply:
Of course! I'll do that, thanks! 👍